legend-engine-service-post-validation-runner/src/main/java/org/finos/legend/engine/service/post/validation/runner/ServicePostValidationRunner.java

```diff
@@ -54,7 +54,7 @@
 {
     private final MutableList<CommonProfile> profiles;
 
-    protected static final PlanExecutor planExecutor = PlanExecutor.newPlanExecutor(Relational.build(), ServiceStore.build(), InMemory.build());
+    protected static final PlanExecutor planExecutor = PlanExecutor.newPlanExecutor(Relational.build(), new ServiceStoreExecutorBuilder().build(), InMemory.build());
```

Comment:
lets introduce config

Reply:
We build a default configuration with an empty credentialProviderProvider if no config is passed